src/Security/Authentication/Negotiate/src/NegotiateHandler.cs

```diff
@@ -57,7 +57,7 @@ protected new NegotiateEvents Events
         /// <returns></returns>
         protected override Task<object> CreateEventsAsync() => Task.FromResult<object>(new NegotiateEvents());
 
-        private bool IsHttp2 => string.Equals("HTTP/2", Request.Protocol, StringComparison.OrdinalIgnoreCase);
+        private bool IsSupportedProtocol => HttpProtocol.IsHttp11(Request.Protocol) || HttpProtocol.IsHttp10(Request.Protocol);
```

Comment:
Would this be better implemented as `HttpProtocol.GetVersion(Request.Protocol) <= HttpVersions.HttpVersion11`? As we add protocols I expect range checks to be more common than specific version checks.

Reply:
If anything It should probably be `HttpProtocol.GetVersion(Request.Protocol) < HttpVersions.HttpVersion2`, but I doubt we're ever going to see HTTP/1.2.